src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -11,36 +11,38 @@ public class WorkWithFile {
     private static final String SUPPLY = "supply";
     private static final String BUY = "buy";
     private static final String RESULT = "result";
+    private static final int TYPE_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
-        String data = readFromFile(fromFileName);
+        String[] data = readFromFile(fromFileName);
         String report = createReport(data);
         writeToFile(toFileName, report);
     }
 
-    public String readFromFile(String fromFileName) {
+    private String[] readFromFile(String fromFileName) {
         StringBuilder stringBuilder = new StringBuilder();
         try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));) {
             String line = bufferedReader.readLine();
             do {
-                stringBuilder.append(line).append(COMA);
+                stringBuilder.append(line).append(System.lineSeparator());
                 line = bufferedReader.readLine();
             } while (line != null);
         } catch (IOException e) {
             throw new RuntimeException("Cant read file " + fromFileName, e);
         }
-        return stringBuilder.toString();
+        return stringBuilder.toString().split(System.lineSeparator());
```

Comment:
again you are appending and then splitting, right here, would be better to avoid that,
in the future, you will learn about the lists, and we could have used them here, but we can't, so we will leave it as it is :)

Reply:
thanks)